src/extender.cpp

```diff
@@ -350,10 +350,9 @@ namespace Sass {
     newExtensionsByTarget.insert(std::make_pair(target, newExtensions));
     if (hasExistingExtensions) {
       ExtByExtMap::iterator existingExtensions = extensionsByExtender.find(target);
-      const std::vector<Extension>& extensions = existingExtensions->second;
-      if (!extensions.empty()) {
+      if (!existingExtensions->second.empty()) {
```

Comment:
The bug is here.

`existingExtensions` can be the `end` iterator, in which case it cannot be dereferenced (undefined behaviour).

Reply:
Adding a check for `existingExtensions != extensionsByExtender.end()` makes the test suite run without crashes for me.

Reply:
That find should not fail since it is in an if(hasExistingExtensions), but can try to add an additional check ..